CONTRIBUTING.md

```diff
@@ -309,6 +309,25 @@ about:
     when someone else is asking for feedback on their code, and universe balances out.
 
 
+Backporting
+-----------
+
+After the original PR was merged into master, backporting is done only
+when necessary - the fix either needs to be present in a release branch
+(e.g. a security issue) or on request.
+If the backport is trivial and can be cherry-picked automatically,
+just request a "Needs backport" tag for the original PR.
```

Comment:
```suggestion
just request a "Needs backport" label for the original PR
```

Reply:
Done in 7c5ce792ef498598bd57780d42938139dbcd58e7 and will be squashed in the end.